Multisig/UI/ClaimToken/Select Guardian/GuardianListViewController.swift

```diff
@@ -72,7 +72,7 @@ class ChooseGuardianViewController: LoadableViewController {
             filteredGuardians = guardians.filter { guardian in
                 return guardian.name?.lowercased().contains(searchTerm) ?? false ||
                 guardian.ensName?.lowercased().contains(searchTerm) ?? false ||
-                guardian.address == Address(searchTerm)
+                guardian.address.hexadecimal.lowercased().contains(searchTerm)
```

Comment:
If I understand the documentation correctly, then the result of hexadecimal will always be lowercase.